data-ai/anomaly-detection.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Anomaly Detection with Azure Databricks
+category: Machine Learning, Advanced Analytics, AI, Data
```

Comment:
Category must be same as the folder it is placed in, so in this case "data-ai", all lower case. You can not place content into multiple categories, that is what tags are for